test/package.json

```diff
@@ -12,8 +12,11 @@
   "license": "ISC",
   "dependencies": {
     "@smartbear/visualtest-cypress": "file:..",
-    "cypress": "^12.17.3",
     "mochawesome": "^7.1.3",
-    "mochawesome-merge": "^4.3.0"
+    "mochawesome-merge": "^4.3.0",
+    "semver": "^7.5.4"
+  },
+  "devDependencies": {
```

Comment:
dev dependcies isn't required as this doesn't go to NPM and is just for testing locally